release/ray_release/tests/test_test.py

```diff
@@ -67,14 +75,17 @@ def test_get_ray_image():
         == "rayproject/ray-ml:123456-py38-gpu"
     )
     os.environ["BUILDKITE_BRANCH"] = "releases/1.0.0"
-    assert _stub_test({}).get_ray_image() == "rayproject/ray:1.0.0.123456-py37"
+    assert (
+        _stub_test({"cluster": {"byod": {}}}).get_ray_image()
+        == "rayproject/ray:1.0.0.123456-py37"
+    )
 
 
 def test_get_anyscale_byod_image():
     os.environ["BUILDKITE_BRANCH"] = "master"
     os.environ["BUILDKITE_COMMIT"] = "1234567890"
     assert (
-        _stub_test({}).get_anyscale_byod_image()
+        _stub_test({"python": "3.7", "cluster": {"byod": {}}}).get_anyscale_byod_image()
```

Comment:
why is this "3.7"?

Reply:
it can be any version, mostly so that I don't have to change the output